plugins/generator-1.19.4/forge-1.19.4/utils/triggers.java.ftl

```diff
@@ -185,6 +185,133 @@
 }
 </#macro>
 
+<#-- Block-related triggers -->

```

Comment:
Please recheck this file carefully, you are redefining macros already defined

Reply:
Done!

Reply:
P.S. I placed armor triggers at the end of the file, as it is in 1.19.2 template

Reply:
Keep order as it is, as armor triggers are better fitted below item triggers than below block triggers, so file needs no changes in context of this PR

Reply:
Ok it's done